packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -268,7 +268,12 @@ export function makeRumPublicApi(
   }
 
   function overrideInitConfigurationForBridge<C extends InitConfiguration>(initConfiguration: C): C {
-    return { ...initConfiguration, applicationId: 'empty', clientToken: 'empty', sampleRate: 100 }
+    return {
+      ...initConfiguration,
+      applicationId: '00000000-aaaa-0000-aaaa-000000000000',
```

Comment:
Need a valid application id for the e2e format test. The issue wasn't there with the FF because the `initRum` => `canUseEventBridge` => `isExperimentalFeatureEnabled('event-bridge')` always returned false. It wasn't really problematic though because it was only to override some init config options.

Reply:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Maybe change the e2e test instead?

Reply:
I still prefer our event to match the RUM event format, as it serve as a doc to communicate with others
Wdyt?

Reply:
Ok, makes sense!